src/main/java/cubicchunks/server/chunkio/async/forge/AsyncColumnIOProvider.java

```diff
@@ -40,11 +40,10 @@
 
 	@Override void runSynchronousPart() {
 		if (column != null) {
+			//TODO: ChunkDataEvent.Load and Save
+			//MinecraftForge.EVENT_BUS.post(new ChunkDataEvent.Load(column, this.nbt)); // Don't call ChunkDataEvent.Load async
 			column.setLastSaveTime(this.colInfo.world.getTotalWorldTime());
 		}
-
-		// We run the callback even if nothing was loaded, to notify waiting processes that the load failed
-		runCallbacks();
```

Comment:
I realized that a few other parts of code are wrong, and separating runnign callbacks from sync part may not be needed. I will fix that.